src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -58,7 +58,7 @@ public void parse_validArgsWithClassId_returnsFindCommand() {
     }
 
     @Test
```

Comment:
All of your testcases should follow the underscore naming scheme, not camelcase

Reply:
fixed